src/main.c

```diff
@@ -2666,14 +2735,22 @@ static int on_config_listen_element(h2o_configurator_command_t *cmd, h2o_configu
             return -1;
         if (rcvbuf_node != NULL && h2o_configurator_scanf(cmd, *rcvbuf_node, "%u", &stream_rcvbuf) != 0)
             return -1;
+        if (reconnect_interval_node != NULL && h2o_configurator_scanf(cmd, *reconnect_interval_node, "%lu", &reconnect_interval) != 0)
```

Comment:
Scanf designator for uint64_t is `"%" SCNu64`.

Reply:
Addressed in https://github.com/h2o/h2o/pull/3405/commits/a512ac174240ded47c5ed5546882a017ef22649e